documentation/asciidoc/computers/configuration/uart.adoc

```diff
@@ -58,47 +84,51 @@ dtoverlay=uart1,txd1_pin=32,rxd1_pin=33
 
 On the Raspberry Pi, one UART is selected to be present on GPIO 14 (transmit) and 15 (receive) - this is the primary UART. By default, this will also be the UART on which a Linux console may be present. Note that GPIO 14 is pin 8 on the GPIO header, while GPIO 15 is pin 10.
 
+On the Raspberry Pi 5, the primary UART appears on the Debug header.
+
 === Secondary UART
 
 The secondary UART is not normally present on the GPIO connector. By default, the secondary UART is connected to the Bluetooth side of the combined wireless LAN/Bluetooth controller, on models which contain this controller.
 
 === Primary and Secondary UART
 
-The following table summarises the assignment of the first two UARTs:
+The following table summarises the assignment of UARTs on various Raspberry Pi devices:
 
 |===
-| Model | first PL011 (UART0) | mini UART
+| Model | Primary/console | Secondary/Bluetooth
 
 | Raspberry Pi Zero
-| primary
-| secondary
+| UART0
+| UART1
 
 | Raspberry Pi Zero W / Raspberry Pi Zero 2 W
-| secondary (Bluetooth)
-| primary
+| UART1
+| UART0
 
 | Raspberry Pi 1
-| primary
-| secondary
+| UART0
+| UART1
 
 | Raspberry Pi 2
-| primary
-| secondary
+| UART0
+| UART1
 
 | Raspberry Pi 3
-| secondary (Bluetooth)
-| primary
+| UART1
+| UART0
 
 | Compute Module 3 & 3+
-| primary
-| secondary
+| UART0
+| UART1
 
 | Raspberry Pi 4
-| secondary (Bluetooth)
-| primary
-|===
+| UART1
+| UART0
 
-NOTE: The mini UART is disabled by default, whether it is designated primary or secondary UART.
+| Raspberry Pi 5
+| UART10
```

Comment:
What is UART10? it is not mentioned in the table of UART0 -> UART4 above. This isn't super clear. Nor is it clear how this relates to settings like /boot/cmdline.txt - is this table defaults?

Reply:
The information is there, just spread across multiple sections:

> On the Raspberry Pi 5, the primary UART appears on the Debug header.

and:

| Model | Primary | Secondary |
|--------|--------|--------|
| Raspberry Pi 5 | UART10 | dedicated UART |

and:

> /dev/ttyAMA10 | Raspberry Pi 5 Debug UART

and:

> On the Raspberry Pi 5, /dev/serial0 is a symbolic link that points to /dev/ttyAMA10

Reply:
ok maybe to explain my confusion a bit more. I think firstly it's not clear that /dev/ttyAMAx refers to UARTx. And I'm still not clear which UART from the PL011 is being used for UART10 - or is it another UART altogether? "Raspberry Pi 5 has an additional four PL011s, which are disabled by default:" doesn't mention UART10 so it might be handy to do that?

Reply:
All the Pi 5 UARTs we use are PL011s. UARTs 0-4 are in the RP1 I/O chip, UART10 is in the BCM2712 SoC.